backend/api/simulation.py

```
@@ -3301,77 +3274,40 @@ async def get_simulation_grading(
-    # Compose overall grading using OpenAI
+    # Compose overall grading using RAG-enabled grading agent
     scenario = db.query(Scenario).filter(Scenario.id == scenario_id).first()
     learning_outcomes = scenario.learning_objectives if scenario else []
     if isinstance(learning_outcomes, str):
         learning_outcomes = [learning_outcomes]
-    all_user_responses = [msg["content"] for msgs in user_msgs_by_scene.values() for msg in msgs]
+    
     # Calculate average scene score
     scene_scores = [scene["score"] for scene in scene_feedback]
     if scene_scores:
         overall_score = int(round(sum(scene_scores) / len(scene_scores)))
```

Ruff (0.14.1)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)